AppCenterCrashes/AppCenterCrashes/MSACCrashes.mm

```diff
@@ -209,15 +209,15 @@ @implementation MSACCrashes
 
 #pragma mark - Public Methods
 
-+ (NSString *)trackError:(NSError *)error
++ (NSString *)trackError:(NSError *_Nonnull)error
```

Comment:
It doesn't restrict you from passing `nil` here in objective-c, but only generates a warning if I pass `nil`.
Check https://developer.apple.com/swift/blog/?id=25.
I think we should check with `if` as well, shouldn't we?

Reply:
Decided to avoid adding additional conditions. Annotation should be enough.